runtime/crab/src/migration.rs

```diff
@@ -68,6 +68,52 @@ fn migrate() -> frame_support::weights::Weight {
 		}
 	});
 
+	let mut w = 26;
+
+	w += migration_helper::PalletCleaner {
+		name: b"BridgeKusamaGrandpa",
+		values: &[
+			b"RequestCount",
+			b"InitialHash",
+			b"BestFinalized",
+			b"ImportedHashesPointer",
+			b"CurrentAuthoritySet",
+			b"PalletOwner",
+			b"PalletOperatingMode",
+		],
+		maps: &[b"ImportedHashes", b"ImportedHeaders"],
+	}
+	.remove_all();
+	w += migration_helper::PalletCleaner {
+		name: b"BridgePolkadotGrandpa",
+		values: &[b"PalletOwner", b"PalletOperatingMode"],
+		maps: &[b"ParasInfo", b"ImportedParaHeads", b"ImportedParaHashes"],
```

Comment:
The migration value and maps seems differ from the grandpa pallet storage layout.